pkg/scripts/generate-variables.sh

```diff
@@ -1,5 +1,7 @@
 # This script will generate/override the `./pkg/tfstate/tfstate-locals.tf` and `./terraform-backend.tf`
 
+source './pkg/scripts/common.sh'
```

Comment:
This will break if the generate-variables script call from any folder other than root. However, I think we can make this script a bit simple to fix this. Actually we don't need to pass the root path. We either run this script dirctly from terminal, or is run by install.sh and uninstall.sh. 

The script path can be found by this script:

```
if [ "${0##*/}" == "cleanup.sh" ]; then
  # the script ran directly from terminal
  ROOT_PATH=$(cd $(dirname "${0}")/../..; pwd)
else
  # the script called by install.sh or uninstall.sh
  ROOT_PATH=$(cd $(dirname "${0}"); pwd)
fi
SCRIPT_PATH="${ROOT_PATH}/pkg/scripts"

source "${SCRIPT_PATH}/common.sh"
```